configs/pytorch.json

```diff
@@ -1,8 +1,14 @@
 {
   "index_name": "pytorch",
   "start_urls": [
+    "https://pytorch.org",
+    "https://pytorch.org/index.html",
     "https://pytorch.org/get-started/",
-    "https://pytorch.org/get-started/locally/"
+    "https://pytorch.org/get-started/locally/",
+    "https://pytorch.org/features",
+    "https://pytorch.org/ecosystem",
+    "https://pytorch.org/blog/",
+    "https://pytorch.org/resources"
```

Comment:
Since this page is gathering information available somewhere else on the site (getting started, tutorials, ...).
We will not scrap this page.

Reply:
Similar to `ecosystem` - But we own that page itself and the words on it. For example we own the words in the boxes before they are linked externally. Isn't that ok?